src/main/java/ai/zuva/fields/FieldAccuracy.java

```diff
@@ -0,0 +1,13 @@
+package ai.zuva.fields;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class FieldAccuracy {
+    @JsonProperty("f_score")
+    public float fScore;
+
+    public float precision;
+    public float recall;
```

Comment:
Is there a reason that some of these properties have the respective @JsonProperty annotation and some don't?

Reply:
Also just to note, in the future with the next release example count should be added

Reply:
> Is there a reason that some of these properties have the respective @JsonProperty annotation and some don't?

It looks like you only need to define the `JsonProperty` if the names don't match.

Reply:
Ah, that makes sense

Reply:
That's the theory. I may have sprinkled some unnecessary ones in there when Jackson wasn't doing what I wanted and I couldn't figure out why.